### DISCUSSION TITLE
[Enhancement] [table] Fixed header and scroll bar with the page

### DISCUSSION THREAD

**Discussion by guicode** [element-plus/element-plus#19216] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

For long table，need to scroll to view the header and scroll bar，then you can now set the fixed header and scroll bar to follow the page.
demo: https://www.antdv.com/components/table#components-table-demo-sticky

### Additional comments

_(empty)_

**Comment by Dsaquel** [comment]
element plus already have this feature by default if you set an `height`: [docs](https://element-plus.org/en-US/component/table#table-with-fixed-header)
